src/couch_replicator_manager.erl

```diff
@@ -338,16 +364,18 @@ terminate(_Reason, State) ->
 
 code_change(1, State, _Extra) ->
     {ok, erlang:append_element(State, [node() | nodes()])};
+code_change(2, State, _Extra) ->
+    {ok, erlang:append_element(State, make_ref())};
```

Comment:
You either need to update the code change for vsn 1 to add this as well or remove this. We haven't been doing hot code upgrades and I'd argue against making it half way cause this would blow up if our version is old enough.

Alternatively delete the code_change for 1 so its obvious we're not supporting that far of a change.